storage-model/src/main/java/no/unit/nva/publication/storage/model/daos/ResourceDao.java

```diff
@@ -22,11 +27,24 @@ public ResourceDao(Resource resource) {
         this.data = resource;
     }
 
+    public static ResourceDao queryObject(UserInstance userInstance, SortableIdentifier resourceIdentifier) {
+        Resource resource = Resource.emptyResource(
+            userInstance.getUserIdentifier(),
+            userInstance.getOrganizationUri(),
+            resourceIdentifier);
+        return new ResourceDao(resource);
+    }
+
     public static String constructPrimaryPartitionKey(URI customerId, String owner) {
         return String.format(PRIMARY_KEY_PARTITION_KEY_FORMAT, Resource.TYPE,
             orgUriToOrgIdentifier(customerId), owner);
     }
 
+    @JsonIgnore
+    public static String getOrderedContainedType() {
```

Comment:
static method version for the "getOrderedType". Static method is necessary for queries, so that we do not generate empty objects to get this value and the non-static method is necessary for implementing the interfaces.